src/lightning/pytorch/core/datamodule.py

```diff
@@ -32,27 +32,34 @@ class LightningDataModule(DataHooks, HyperparametersMixin):
 
     Example::
 
-        class MyDataModule(LightningDataModule):
-            def __init__(self):
-                super().__init__()
+        import lightning as L
+        import torch.utils.data as data, torchvision as tv
+
+        class MyDataModule(L.LightningDataModule):
             def prepare_data(self):
-                # download, split, etc...
+                # download, IO, etc. Useful with shared filesystems
                 # only called on 1 GPU/TPU in distributed
+                tv.datasets.MNIST(".", download=True)
```

Comment:
negative... add a demo dataset (i think we have one)
and import it here

```python
return DataLoader(L.demo.Dataset(5000))
```

Reply:
Done! Used a simple Python range.